data/lua/wml/find_path.lua

```diff
@@ -28,23 +28,30 @@ function wesnoth.wml_actions.find_path(cfg)
 
 	if not cfg.check_visibility then viewing_side = 0 end -- if check_visiblity then shroud is taken in account
 
-	local locations = wesnoth.get_locations(filter_location) -- only the location with the lowest distance and lowest movement cost will match. If there will still be more than 1, only the 1st maching one.
+	-- only the location with the lowest distance and lowest movement cost
+	-- will match. If there will still be more than 1, only the 1st maching one.
```

Comment:
May as well fix the typo here while you're at it.

Reply:
done.